### PR TITLE
feat: document Compose UI redaction

### PR DESCRIPTION
**Pull request by lassana**

### CODE REVIEW

File: sdk-features/redact-sensitive-data.md [hunk 1]
```diff
@@ -180,6 +180,34 @@ import { Redacted } from 'cobrowse-sdk-react-native';
...
+```
+dependencies {
+    // ... other dependencies ...
+    implementation 'io.cobrowse:cobrowse-sdk-android:2.+'
+    implementation 'io.cobrowse:cobrowse-sdk-android-compose-ui:2.+'
+}
+```
+
+{% hint style="info" %}
+You are supposed to use the same version of the Cobrowse.io SDK and Compose UI redaction archifacts. Using different versions of Cobrowse.io SDK artifacts is not supported.
```
Comment (headlessme) [inline sdk-features/redact-sensitive-data.md:196]: typo: archifacts -> artifacts

Comment (headlessme) [reply sdk-features/redact-sensitive-data.md:196]: Maybe also change "You are supposed to use " to "You are required to use "

Comment (lassana) [reply sdk-features/redact-sensitive-data.md]: Adjusted.

**Comment by steprescott** [comment]
@headlessme I can't yet see this live on https://docs.cobrowse.io/sdk-features/redact-sensitive-data.

**Comment by headlessme** [comment]
@jracabado are you expecting the gitbook GitHub sync to work now? Looks like it hasn't picked up this change yet

**Comment by snyderj** [comment]
@headlessme @jracabado 
Here's what gitbook is showing (right sidebar). If it's still not live, maybe try a small update through the gitbook UI to trigger a re-deploy? 

Screenshot 2024-04-15 at 8 01 19 AM

**Comment by jracabado** [comment]
Hi,

Not sure if there was a delay but I see now that these changes are visible:

[image: image.png]


--



João Acabado
Solutions Engineer @ Cobrowse.io
*W*: https://cobrowse.io
<https://www.linkedin.com/company/cobrowse-io>
<https://twitter.com/cobrowseio>

**Pull request merged**